packages/typo3-docs-theme/resources/config/typo3-docs-theme.php

```diff
@@ -61,6 +61,8 @@
         ->arg('$languageAliases', [
             'none' => 'plaintext',
             'text' => 'plaintext',
+            'xlf' => 'xml',
```

Comment:
the third one is already in place

Reply:
I like to avoid an alias of xlf to xml. xlf is xml, so no need to alias it.

"none" and "text" are defined for backward-compatibility reasons, as this is used by the Python-based pygments code highlighter. We do not want to change this all at once.

Reply:
I go with Garvin that adding sensible aliases makes the life for users easier.

Reply:
So, you want to add an alias von xlf as people don't recognise this as xml. But you want to force users to use WSL2 for rendering?

IMHO an alias should be the last resort. Personally, I want to get rid of them as a remain of the pygments thing.

Reply:
How is the language information displayed when no caption is defined?

Reply:
see bellow